docs/DeveloperGuide.md

```diff
@@ -391,31 +380,101 @@ For all use cases below, refer to the [User Guide](UserGuide.md) for more detail
     - **1c2.** Use case resumes from step 1.
 - **1d.** Person is a client of a wedding
     - **1d1.** Error message is displayed in result display.
+    - **1d2.** User do <u>UC9 - Delete a wedding</u>
+    - **1d3.** User redo this use case.
+        - Use case ends.
+
+#### **Use Case: UC5 - Edit a Contact**
+1. Wedding Organizer edits a contact using the edit command.
+2. Contact details are updated.
```

Comment:
can specify dup contacts with name containing input name... to be consistent w the other use cases